cypress/support/step-definitions/common-steps.js

```diff
@@ -538,7 +538,11 @@ When('I press Home on focused element', () => {
 });
 
 When('I press uparrow on focused element', () => {
-  cy.focused().trigger('keydown', { keyCode: 38, which: 38 });
+  cy.focused().trigger('keydown', { key: 'ArrowUp', keyCode: 38, which: 38 });
```

Comment:
and probably it would be better to use switch here?

Reply:
fixed. using switch.